app/helpers/application_helper.rb

```diff
@@ -60,4 +60,10 @@ def cancel_link_text
   def desktop_device?
     !BrowserCache.parse(request.user_agent).mobile?
   end
+
+  def selfie_status_from_response(client_response)
+    return client_response.selfie_status if client_response.respond_to?(:selfie_status)
+
+    :not_processed
+  end
```

Comment:
why was this helper added here? this is a very generic helper, but this is a specific method? What if we made a mixin just for this?